resources/assets/components/SignupCard/index.js

```diff
@@ -1,92 +1,66 @@
 import React from 'react';
-import { map } from 'lodash';
-import PropTypes from 'prop-types';
-import './signup-card.scss';
 import PostTile from '../PostTile';
-import { RestApiClient } from '@dosomething/gateway';
-import { extractPostsFromSignups } from '../../helpers';
 
-class SignupCard extends React.Component {
-  render() {
-    const signup = this.props.signup;
-    const campaign = this.props.campaign;
-    const gallerySize = 4;
+import './signup-card.scss';
 
-    const extraPostCount = signup.posts.data.length - gallerySize;
+const POST_GALLERY_SIZE = 4;
 
-    const posts = signup.posts.data
-      .slice(0, gallerySize)
-      .map(post => <PostTile key={post.id} post={post} />);
+const SignupCard = ({ signup }) => {
+  const campaign = signup.campaign || {};
 
-    const signupUrl = `/signups/${signup.id}`;
+  const extraPostCount = signup.posts.length - POST_GALLERY_SIZE;
+  const posts = signup.posts.slice(0, POST_GALLERY_SIZE).map(post => (
+    <li key={post.id}>
+      <PostTile post={post} />
+    </li>
+  ));
 
-    return (
-      <article className="container__row signup-card">
-        <a href={signupUrl}>
-          <div className="container__block -half">
-            <div className="container__row">
-              <h2 className="heading">
-                {campaign ? campaign.internal_title : signup.campaign_id}
-              </h2>
-              <h4 className="heading">Campaign ID: {signup.campaign_id}</h4>
-              {campaign ? (
-                <h4 className="heading">
-                  Campaign Start Date: {campaign.start_date}
-                </h4>
-              ) : null}
-            </div>
-            <div className="container__row">
-              <h4 className="heading">Why Statement</h4>
-              <p>{signup.why_participated}</p>
-            </div>
+  return (
+    <article className="container__row signup-card">
+      <a href={`/signups/${signup.id}`}>
+        <div className="container__block -half">
+          <div className="container__row">
+            <h2 className="heading">
+              {campaign.internalTitle || 'Unknown Campaign'}
+            </h2>
+            <h4 className="heading">Campaign ID: {campaign.id}</h4>
+            <h4 className="heading">
+              Campaign Start Date: {campaign ? campaign.startDate : '???'}
```

Comment:
Why do we condition the `campaign`s existence here as opposed to [above](https://github.com/DoSomething/rogue/pull/915/files#diff-ff380d564bdc83de86dff5ffd7dcd30bR24) re the `campaign.internalTitle`?

Reply:
Oh, no particular reason. I'll swap both of these to use the `|| 'fallback'` pattern.

Reply:
Updated in 1ab4eb09.